xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -976,6 +976,7 @@ core.menu.translate=Translate
 core.menu.translate.hint=Translate this page in {0}
 core.menu.translate.details=You''re viewing the {0} translation of this page (its original version) because this page has not been translated in {1} (the current locale) yet.
 core.menu.edit=Edit
+core.menu.edit.dropdown.hint=Select an edit mode to use.
```

Comment:
I wonder if something like "Edit modes" wouldn't be more consistent with the short submenu headings in the more actions menu (that just has "Manage", "Actions", "Viewers").

Reply:
You're right, and the use of a verb here is not very appropriate for this kind of label. :+1: 
Addressed in https://github.com/xwiki/xwiki-platform/pull/2190/commits/16fc7ba863d7c5652e50a7a4ddbb3f6e8d9ca15e